docs/architecture/adr-006-hermes-v0.2-usecases.md

```diff
@@ -0,0 +1,269 @@
+# ADR 006: Hermes v0.2.0 Use-Cases
+
+## Changelog
+* 16.02.2021: Proposed.
+
+## Context
+
+One major problem with planning for the evolution of Hermes is that presently
+there is insufficient clarity regarding its requirements.
+It is not known who are the typical Hermes users (is it human operators or
+automated pipelines?), and what are their primary use-cases.
+
+This ADR proposes a few use-cases that seem interesting from the point
+of view of a general target base of users, and which will
+hopefully be a subset of the requirements of (any) future users.
+
+Three elements that provide further context for this discussion are:
+
+1. Hermes is still at an early stage of implementation, so these use-cases are
+   not set in stone.
+
+2. Some concrete use-cases are starting to emerge ([#628][628]), which Hermes
+   v0.1.0 either does not cover altogether, or covers poorly (e.g., because of
+   inconsistent UX), thus informing this proposal.
+
+3. Hermes is one of _three_ relayer binaries that are being developed roughly in
+parallel. The other two are being developed in Go and Typescript, 
+respectively (see the [references](#references) section).
+In this context, it is plausible that Hermes will focus on performance,
+robustness, and richness of features on a longer term.
+
+## Decision
+
+This is a summary of the use-cases (commands) discussed in the rest of this ADR.
+Note that the commands below omit the binary name `hermes` , to keep the command
+length to a minimum.
+
+To create and update a client:
+- `create client <src-chain-id> <dst-chain-id>`
```

Comment:
Oh, just noticed we have src chain first and then dst chain. Until now we used the other order. First we specify the destination and then the source.

Reply:
Actually I would propose we stop using the `src` and `dst` for these commands. With the exception of the client, all the others involve multiple transactions so using src/dst doesn't make sense anymore.
Maybe sat `a` and `b` or something else. For connection and channel we can say that `open-init` is always sent to `a` (if that's the case)

Reply:
I still like a bit more:
"create client on X for chain Y" -> `create client <chain-x> <chain-y>` 
instead of
"create client for Y on chain X" -> `create client <chain-y> <chain-x>` 
But I can live with either :)

Reply:
- For connection & channel: I like the `a` `b` distinction and I think we should make more effort to use this naming scheme consistently. Will use this from now.
- For client: cool to use `destination` before `source`! As a more general comment, I would find it more intuitive and accurate to use the naming scheme `host`/`target` instead of `destination`/`source`, but I don't insist on it.

Reply:
Introduced the scheme with "a" and "b" 92ffdd1e.